cypress.config.ts

```diff
@@ -15,7 +15,8 @@ const config = defineConfig({
         experimentalRunAllSpecs: true,
         requestTimeout: 10000,
         responseTimeout: 30000,
-        pageLoadTimeout: 60000
+        pageLoadTimeout: 60000,
+        defaultCommandTimeout: 10000
```

Comment:
this is the original approach we had, but we had issues with the timeouts not being applied correctly 🤔 
I'm not sure if it was due to an unrelated error. I'm nervous to commit, but I see the tests are passing and I think it's much cleaner